README.md

```diff
@@ -4,6 +4,15 @@
 
 ## Installation
 
+This documentation is built with Sphinx, a Python documentation generator. You will thus need to install a [Python runtime](https://www.python.org/downloads/) to build it. The version to install is specified in the `runtime.txt` file.
+
+In order to avoid conflicting dependencies with other projects on your local machine, it is recommended to install its dependencies in a virtual environment. To create a virtual environment, run:
```

Comment:
I'm not sure about that: either you know what is a venv and you don't need that precision, or you don't and this added information is not enough and might be more confusing. We could add something like:

```suggestion
In order to avoid conflicting dependencies with other projects on your local machine, it is recommended to install its dependencies in a virtual environment. To create a virtual environment and install all dependencies in a local `.venv` directory, run:
```

…but I wonder how useful that really is to someone who just wants to contribute to the doc!

Reply:
I'm not sure about that: either you know what is a venv and you don't need that precision, or you don't and this added information is not enough and might be more confusing. We could add something like:

```suggestion
In order to avoid conflicting dependencies with other projects on your local machine, it is recommended to install its dependencies in a virtual environment. To create a virtual environment and install all dependencies in a local `.venv` directory, run:
```

…but I wonder how useful that really is to someone who just wants to contribute to the doc!